openai-assistant/pom.xml

```diff
@@ -16,7 +16,7 @@
     <repository>
       <id>ivy.nightly.release</id>
       <layout>p2</layout>
-      <url>https://p2.axonivy.com/p2/nightly-11.2/</url>
+      <url>https://p2.axonivy.com/p2/nightly/</url>
```

Comment:
while it is true, that 11.2 is still compliant ... the 'dev' job is here, that the latest 'core' is also compliant. so we should keep it testing the 'nightly'